ts/src/upbit.ts

```diff
@@ -1759,17 +1759,16 @@ export default class upbit extends Exchange {
             }
             params = this.omit (params, [ 'network' ]);
             request['net_type'] = network;
-            method += 'Coin';
             request['currency'] = currency['id'];
             request['address'] = address;
             if (tag !== undefined) {
                 request['secondary_address'] = tag;
             }
             params = this.omit (params, 'network');
+            response = await this.privatePostWithdrawsCoin (this.extend (request, params));
         } else {
-            method += 'Krw';
+            response = await this.privatePostWithdrawsKrw (this.extend (request, params));
```

Comment:
@sc0Vu you said that the static tests were converting the edits but you've changed `withdraw` and I can only see a static test for loadMarkets 🤔

Reply:
@carlosmiei sor, I missed that one. The pr was updated!